.github/workflows/issue.yml

```
@@ -15,10 +15,12 @@ jobs:
         with:
           issue-number: ${{ github.event.issue.number }}
           body: |
-            Thanks for the issue! We'll take your request into consideration and follow up if we decide to tackle this issue.
+            Thanks for the issue, we will take it into consideration! Our team of engineers is extremely busy working on many types of features, please give us time to get back to you.  
```

non blocking: I wonder if `extremely` can sound discouraging to contributors

```suggestion
            Thanks for the issue, we will take it into consideration! Our team of engineers is busy working on many types of features, please give us time to get back to you.
```